### ISSUE TITLE
Support Bitbucket snippets

### ISSUE THREAD

**Issue by miloskroulik** [Gisto/Gisto#177]
It would be great to support also Bitbucket snippets, which is now possible, [because Bitbucket API now supports working with them](http://blog.bitbucket.org/2015/09/08/snippets-api-wiki-edits-restrictions-and-more/).

**Comment by morsdyce** [comment]
Currently our application is heavily factored into the GitHub ecosystem, it will take allot of effort to get that working.

We are planning an overview of the project in the coming months where we will improve existing code and might add an abstraction so you may use either github or bitbucket.

Thanks

**Comment by miloskroulik** [comment]
Thanks a lot for consideration.

**Comment by pacificera** [comment]
A year later, this probably isn't happening.

**Comment by sanusart** [comment]
Not at all. We just not pushing on development lately. Waiting to start a complete rewrite.
